proof-producer/libs/assigner/CMakeLists.txt

```diff
@@ -25,13 +23,13 @@ foreach(PROTO_FILE ${PROTO_FILES})
     list(APPEND GENERATED_HEADERS ${PROTO_HDR})
 endforeach()
 
-add_library(proof_generatorAssigner STATIC
+add_library(proof_generatorAssigner
             ${PROTO_SRC}
```

Comment:
Why?

Reply:
Now it's possible to build either static or dynamic libraries depending on `BUILD_SHARED_LIBS` flag. But it's still static for all the derivations, due to the issue I described above